### PR TITLE
default policy for slaDrainHosts

### PR DESCRIPTION
**Pull request by lenhattan86**
default policy for slaDrainHosts when users do not specify the policy.

Tests Done:
- unit test.
- integration test.

### CODE REVIEW

File: realis_admin.go [hunk 1]
```diff
@@ -59,6 +59,21 @@ func (c *Client) SLADrainHosts(policy *aurora.SlaPolicy, timeout int64, hosts ..
 		return nil, errors.New("no hosts provided to drain")
 	}
 
+	if policy == nil ||
+		(policy.PercentageSlaPolicy == nil &&
```
[the remark this answers is not in the document]
Comment (lenhattan86) [reply realis_admin.go:63]: good point. will fix.

[the remark this answers is not in the document]
Comment (ridv) [reply realis_admin.go:63]: I thought about this some more, it has a bug as it is. This should work thought:

```suggestion
		switch {
	case policy == nil:
		policy = &defaultSlaPolicy
	case policy.Percentage != nil:
	case policy.CountSlaPolicy != nil:
	case policy.CoordinatorSlaPolicy != nil:
	default:
		policy = &defaultSlaPolicy
		fmt.Printf("Warning: start draining with default sla policy %v", policy)
	}
```

[the remark this answers is not in the document]
Comment (lenhattan86) [reply realis_admin.go:63]: a bit too messy, tried with another way.

**Comment by ridv** [comment]
You should merge changes from master into this PR cause they snuck into it.

**Pull request merged**